certora/specs/Base.spec

```diff
@@ -78,7 +78,7 @@ methods {
 
     // OperatorsRegistryV1
     function _.reportStoppedValidatorCounts(uint32[], uint256) external => DISPATCHER(true);
```

Comment:
If the operators registry contract is removed from the scene, that dispatch summary will cause vacuity. Consider commenting-out the summary or replace it with NONDET.